crates/aptos/src/governance/mod.rs

```diff
@@ -475,6 +467,6 @@ impl CliCommand<&'static str> for GenerateUpgradeProposal {
         let package = BuiltPackage::build(package_path, options)?;
         let release = ReleasePackage::new(package)?;
         release.generate_script_proposal(account, output)?;
-        Ok("success")
+        Ok(())
```

Comment:
Why?

Reply:
This was following non-standard outputs.  I've fixed it accordingly